django/http/request.py

```diff
@@ -305,6 +305,12 @@ def body(self):
             self._stream = BytesIO(self._body)
         return self._body
 
+    @property
+    def preferred_language(self):
+        """Returns the client's language in order of priority from the "Accept-Language" header"""
+        languages = [_.strip() for _ in self.headers.get('Accept-Language', '*').split(',')]
+        return languages[0].split(';')[0]
```

Comment:
This doesn't take into account the weighting that can be present, and assumes that the first item is the preferred one, which is AFAIK only true if _none_ of the options have a q weight (in which case they all have a `q=1`)